rust/src/smb/smb2.rs

```diff
@@ -279,10 +287,18 @@ pub fn smb2_write_request_record<'b>(state: &mut SMBState, r: &Smb2Record<'b>)
     }
     match parse_smb2_request_write(r.data) {
         Ok((_, wr)) => {
-            if (state.max_write_size != 0 && wr.wr_len > state.max_write_size) ||
-               (unsafe { SMB_CFG_MAX_WRITE_SIZE != 0 && SMB_CFG_MAX_WRITE_SIZE < wr.wr_len }) {
+            let max_wr_len = if wr.wr_len as usize - wr.data.len() > more_nbss {
+                // Write record claims more bytes than can be contained in NBSS record...
+                // The server MUST fail the request with STATUS_INVALID_PARAMETER.
+                // We use tha data available...
```

Comment:
typo `tha`

Reply:
And fixed

By the way, lol : `git grep "tha "`